.github/workflows/unit-tests.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ["2.7"]
+        ruby: ["3.0"]
```

Comment:
We still run Ruby 2.7 in production, so I'd recommend to continue testing that.

Reply:
Ok I will add 2.7 back and get to work on the `Gemfile.dev.rb` sorry I forgot about it with all the 6.14/6.15 stuff going on.